src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorldTest {
+    private User user = new User();
+    private RegistrationService registration = new RegistrationServiceImpl();
```

Comment:
you can use @BeforeAll to initialize this fields (I think it will be ok for this task)